src/overlays/actors/ovl_player_actor/z_player.c

```diff
@@ -27,6 +28,13 @@
 // This is called "adjusted" for now.
 #define PLAYER_ANIM_ADJUSTED_SPEED (2.0f / 3.0f)
 
+// On PAL N64, many speed are increased so Link doesn't feel slow on 50Hz.
+#if !OOT_PAL_N64
+#define ADJUST_SPEED_FOR_FRAME_RATE(speed) (speed)
+#else
+#define ADJUST_SPEED_FOR_FRAME_RATE(speed) ((speed)*60.0f / 50.0f)
+#endif
```

Comment:
Im not 100% sure on my opinion yet, but part of me does want to see both constants with these player and player_lib usages (similar to `FRAME_RATE_DEPENDENT` above)
I want to look at it and go "oh thats the speed value" and not have to go and do some math first

Reply:
I'd prefer seeing both numbers too actually

Reply:
changed to use FRAME_RATE_DEPENDENT everywhere